### DISCUSSION TITLE
향후 계획

### DISCUSSION THREAD

**Discussion by cloudchamb3r** [SWM-Codection/gitea#56] [General]
ㅇ

**Comment by Lieslot** [comment]
- 디스커션 검색 기능
- 디스커션 정렬(좋아요 순, 업데이트 최신 순, 등록 순)
- 디스커션 좋아요 기능
- 디스커션 open/close 프론트
- 마감일 지정
- 담당자 설정
- 디스커션 구독 설정
- 디스커션 파일 뷰어 코드 하이라이트 적용
- 디스커션 코멘트 삭제
- 디스커션 코멘트 답글(넣을지 말지는 생각해봐야 할듯)
- 디스커션 파일 코멘트 프론트 리팩토링 및 기능 완성
- 디스커션 전역 코멘트 프론트 리팩토링 및 기능 완성
- 디스커션 코멘트 인증 관련 예외처리(상세는 기능명세 참조)
- 디스커션 작성/수정 시 바이너리 파일 배제

----MVP 완성 컷
- 전역 코멘트 멘션
- 파일 코멘트 멘션
- 전역 코멘트 반응
- 파일 코멘트 반응
- 디스커션 사용자 멘션
- 대시보드 PR feedback 타임 백엔드
- 대시보드 PR 리뷰 시간 백엔드
- 대시보드 PR 코드 변화량 백엔드
- 대시보드 프론트엔드
- 데이터 파이프라인 구축(모니터링을 위함)

**Reply by cloudchamb3r** [reply]
검색 bleve

**Reply by cloudchamb3r** [reply]
mde -> highlight

**Reply by cloudchamb3r** [reply]
디스커션 코멘트 인증 관련 예외처리(상세는 기능명세 참조)
 => 프론트 쪽 비인가 사용자에 대한 분기 처리

**Reply by cloudchamb3r** [reply]
+ AI Sample Code API 레이트 리밋 
+ 지울거 지우기 i18n (영어 한국어 제외 모두 삭제) 및 한글화
+ project 기능 (칸반) UI 개선
